CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.16)   
 
-project("UnityHubNative")
+project("UnityHubNative" LANGUAGES C CXX OBJCXX)

```

Comment:
Don't explicitly set languages, because it requires all platforms to have support for those languages. For example,the Linux build fails with this set.

Reply:
Can I place this under the if(APPLE) leaving it parameterless in else()? I'm getting this error and according to the internet search it just because the compiler can't tell what language are you compile and guessed wrong.

<img width="843" alt="Screenshot 2024-01-09 at 19 14 12" src="https://github.com/Ravbug/UnityHubNative/assets/6708914/73d217aa-c318-4e2e-a309-725299d058eb">

Reply:
I would use [`enable_language`](https://cmake.org/cmake/help/latest/command/enable_language.html) inside an `if(APPLE)` rather than set it in the `project` command.

Reply:
Ok, thanks, that worked as well!